_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -223,7 +223,7 @@ Now, we see that there are in fact two features in our plot!
 > > nrow(stoneWall_HARV)
 > > ```
 > > Now we can plot the data: 
-> > ```{r harv-stone-wall-map}
+> > ```{r harv-stone-wall-map fig.cap="Map of the stone walls in the study area where each feature is colored differently."}
```

Comment:
```suggestion
> > ```{r harv-stone-wall-map, fig.cap="Map of the stone walls in the study area where each feature is colored differently."}
```